proto/store/policy.proto

```diff
@@ -29,3 +30,29 @@ message Binding {
   // If the condition evaluates to false, then this binding does not apply to the current request. However, a different role binding might grant the same role to one or more of the principals in this binding.
   google.type.Expr condition = 3;
 }
+
+// DataMaskingExceptionPolicy is the whitelist of users who can access sensitive data.
+message DataMaskingExceptionPolicy {
```

Comment:
DataUnmaskingPolicy? DataMaskingExceptionPolicy is fine too.

Reply:
Keep `DataMaskingExceptionPolicy `.